compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -764,10 +810,12 @@ static LogicalResult setMatmulNoPadRootConfig(
   for (auto [index, tileSize] : llvm::enumerate(workgroupTileSizes)) {
     int64_t sz = tileSize;
     if (sz != 0) {
-      sz = getMaxTileSize(/*lb=*/0, /*ub=*/shape[index],
-                          /*maxTileSize=*/sz, vectorSize,
-                          /*allowIncompleteTile=*/vecPreProcStrategy ==
-                              VectorPreProcStrategy::Peeling);
+      sz = getMaxTileSize(
+          /*lb=*/0, /*ub=*/shape[index],
+          /*maxTileSize=*/sz, vectorSize,
+          /*allowIncompleteTile=*/vecPreProcStrategy ==
+                  VectorPreProcStrategy::Peeling ||
+              vecPreProcStrategy == VectorPreProcStrategy::Masking);
```

Comment:
This is getting longer. It'd be easier to read if we declare a `allowIncompleteTile` variable. Something like

```suggestion
      bool allowIncompleteTile = vecPreProcStrategy == VectorPreProcStrategy::Peeling ||
        vecPreProcStrategy == VectorPreProcStrategy::Masking;
      sz = getMaxTileSize(
          /*lb=*/0, /*ub=*/shape[index],
          /*maxTileSize=*/sz, vectorSize, allowIncompleteTile);
```